### PR TITLE
Allow migration file extension to be specified

### PR DESCRIPTION
**Pull request by lightbringer1991**
Add a new option to specify new migration file extension, default to `.coffee`
```bash
node migrations.js --extension=js --create=test1
```

TODO:

- [x] update README

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -50,32 +71,13 @@ Index.prototype.run = function() {
...
-    , '  next();'
-    , '};'
-    , ''
-    , 'exports.down = function(next) {'
-    , '  next();'
-    , '};'
-    , ''
-  ]).join('\n');
+  var supportedExtensions = Object.keys(templates);
+  var extension = supportedExtensions.includes(cli['extension']) ? cli['extension'] : 'coffee';
```
Comment (lightbringer1991) [inline lib/index.js]: not sure if we want default extension to be `js` or `coffee` here.

Comment (adityasabnis) [reply lib/index.js]: js

File: README.md [hunk 2]
```diff
@@ -72,10 +72,11 @@ to the *migrations* executable and run it in a manual way.
...
-    --create  Create empty migration file
-    --count   Migrate particular number of migration
-    --revert  Revert last migration
+    -h, --help      output usage information
+    --up            Migrate up
+    --down          Migrate down
+    --create        Create empty migration file
+    --count         Migrate particular number of migration
+    --revert        Revert last migration
+    --extension     specify a file extension for the new migration file, default to 'coffee'
```
Comment (adityasabnis) [inline README.md]: (`s` -> `S`)pecify a file...

**Comment by lightbringer1991** [comment]
stale, closing

**Pull request closed without merging**